recipe/bld.bat

```diff
@@ -1,22 +1,42 @@
-robocopy . "%PREFIX%" /S /XF bld.bat
-if errorlevel NEQ 1 exit 1
+set "PREFIX_NSIS=%PREFIX%\NSIS"
+robocopy . "%PREFIX_NSIS%" /S /XF bld.bat Docs Examples
+if errorlevel 1 exit 1
 
 cd ..
 if errorlevel 1 exit 1
 
 %PYTHON% "%RECIPE_DIR%\get_plugins.py"
 if errorlevel 1 exit 1
 
-copy "UAC\Plugins\x86-unicode\UAC.dll " "%PREFIX%\Plugins\x86-unicode\"
+copy "elevate\bin.x86-32\elevate.exe" "%PREFIX_NSIS%\Plugins\x86-unicode\"
 if errorlevel 1 exit 1
-copy "Untgz\untgz\unicode\untgz.dll" "%PREFIX%\Plugins\x86-unicode\"
+copy "UAC\U\UAC.dll" "%PREFIX_NSIS%\Plugins\x86-unicode\"
 if errorlevel 1 exit 1
-copy "UnicodePathTest_1.0\Plugin\UnicodePathTest.dll" "%PREFIX%\Plugins\x86-unicode\"
+copy "untgz\Plugins\x86-unicode\untgz.dll" "%PREFIX_NSIS%\Plugins\x86-unicode\"
```

Comment:
Guessing this falls out of changes to the decompression step like respecting the root directory and such.

Reply:
Sorry, I don't quite understand your comment. ("non-native speaker" excuse :wink:)

Reply:
That's probably because I was rambling. 😄 Feel free to ignore this comment.